docs/content/how-to/troubleshooting.md

```diff
@@ -0,0 +1,90 @@
+# Troubleshooting
+
+## How to dump the HostedCluster resources from a management cluster
+
+In order to dump the relevant HostedCluster objects we will need some prerequisites:
+
+- `cluster-admin` access to the management cluster
+- The HostedCluster `name` and the `namespace` where the CR is deployed
+- The Hypershift CLI
+
+Once we have these elements and the shell with the Kubeconfig loaded pointing to the management cluster, we will execute these commands:
+
+```bash
+CLUSTERNAME="samplecluster"
+CLUSTERNS="clusters"
+
+mkdir clusterDump-${CLUSTERNS}-${CLUSTERNAME}
+hypershift dump cluster \
+    --name ${CLUSTERNAME} \
+    --namespace ${CLUSTERNS} \
+    --artifact-dir clusterDump-${CLUSTERNS}-${CLUSTERNAME}
+```
+
+After some time, the output will show something like this:
+
+```bash
+2023-06-06T12:18:20+02:00	INFO	Archiving dump	{"command": "tar", "args": ["-cvzf", "hypershift-dump.tar.gz", "cluster-scoped-resources", "event-filter.html", "namespaces", "network_logs", "timestamp"]}
+2023-06-06T12:18:21+02:00	INFO	Successfully archived dump	{"duration": "1.519376292s"}
+```
+
+This dump contains so much and quite useful information that will help on the troubleshooting procedure.
```

Comment:
s/This dump contains so much and quite useful information that will help on the troubleshooting procedure./This dump contains artifacts that aid in troubleshooting issues with hosted control plane clusters./